react-native/react/login2/index.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Login based on kex2. This will replace login/ when we integrate the FE/BE
+ */
+
+'use strict'
+/* @flow */
```

Comment:
let's just use this ticket to clean this everywhere: https://keybase.atlassian.net/browse/DESKTOP-73